### ISSUE TITLE
Mistake in tensorflow API doc

### ISSUE THREAD

**Issue by LucasMahieu** [tensorflow/tensorflow#19460]
As you can see in the [documentation](https://www.tensorflow.org/api_docs/python/tf/nn/softmax), the 'axis' attribute is not mandatory.
But, its default value is "None" in the function call example, but is '-1' in the description of the 'axis' attribute.

**Comment by asimshankar** [comment]
Yup, as @facaiy mentioned, [`None` is treated as -1](https://github.com/tensorflow/tensorflow/blob/3b85959a57ef2656cdcf2cfac62d68713948ba54/tensorflow/python/ops/nn_ops.py#L1736)
